Business resume homepage/index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="homepage.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Business Resume</title>
+</head>
+<body>
+    <header>
+        <div class="header-left">
+            <div class="header-box">
+                <div class="square">
+                </div>
+                <a href="index.html">Maya Nelson</a>
+            </div>
+            <span>/</span>
+            <a class="title-header" href="index.html">PROJECT MANAGER</a>
+        </div>
+
+        <div class="header-right">
+            <input id="toggle" type="checkbox"></input>
+
+            <label for="toggle" class="hamburger">
+            <div class="top-bun"></div>
+            <div class="meat"></div>
+            <div class="bottom-bun"></div>
+            </label>
+
+            <div class="nav">
+            <div class="nav-wrapper">
+            <nav>
+                <ul>
+                    <li><a href="index.html">About me</a></li>
+                    <li><a href="resume.html">Resume</a></li>
+                    <li><a href="#">Projects</a></li>
+                    <li><a href="contact.html">Contact</a></li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+
+    <main>
+        <div class="main-wrapper">
+            <div class="container">
+                <img src="/Business resume homepage/AdobeStock_230407429.webp" alt="person-pic">
+
+                <h1>Maya Nelson</h1>
+
+                <div class="line"></div>
+
+                <div class="buttons">
+                    <a href="resume.html">RESUME</a>
+                    <a href="">PROJECTS</a>
+                </div>
+
+                <div class="text-container">
+                    <p>PROJECT MANAGER</p>
+                </div>
+
+                <div class="socials">
+                    <ul>
+                    <li><a href="#"><i class="fa-brands fa-facebook-f"></i></a></li>
+                    <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+                    <li><a href="#"><i class="fa-brands fa-linkedin-in"></i></a></li>
+                    <li><a href="#"><i class="fa-brands fa-instagram"></i></a></li>
+                </ul>
+                </div>
+            </div>
+
+            <div class="main-text">
```

Comment:
Is there a semantic html element to section up blocks of a page like this?

Reply:
done